src/components/BlockStyleControls/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react';
+import StyleButton from '../StyleButton';
+
+const BLOCK_TYPES = [
+  { label: 'H1', style: 'header-one' },
+  { label: 'H2', style: 'header-two' },
+  { label: 'H3', style: 'header-three' },
+  { label: 'H4', style: 'header-four' },
+  { label: 'H5', style: 'header-five' },
+  { label: 'H6', style: 'header-six' },
+  { label: 'Blockquote', style: 'blockquote' },
+  { label: 'UL', style: 'unordered-list-item' },
+  { label: 'OL', style: 'ordered-list-item' },
+  { label: 'Code Block', style: 'code-block' },
+];
+
```

Comment:
Constant should be in constants folder, because it's connected to application it should be inside application.js.